### PR TITLE
Resolved the crash when account is not new (Auth)

### PR DESCRIPTION
**Pull request by ezaquarii**
This is a rebased PR https://github.com/nextcloud/android/pull/5061

Signed-off-by: Kilian Périsset <kilian.perisset@infomaniak.com>
Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

### CODE REVIEW

**Comment by Shagequi** [comment]
Hi, I review this one today :)

File: src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java [hunk 1]
```diff
@@ -1513,9 +1510,20 @@ private void updateAuthStatusIconAndText(RemoteOperationResult result) {
             case HOST_NOT_AVAILABLE:
                 mAuthStatusText = getResources().getString(R.string.auth_unknown_host_title);
                 break;
+            case ACCOUNT_NOT_NEW:
+                String errorMessage = getString(R.string.auth_account_not_new);
```
Comment (JorisBodin) [inline src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java]: Why not put directly into mAuthStatusText?

Comment (Shagequi) [reply src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java]: Good point, I applied a quick patch

**Comment by Shagequi** [comment]
@tobiasKaminsky 
@AndyScherzinger 

Good morning :) 
The PR has been tested and approved, may I ask you to watch this please ?
Thanks!

**Comment by Shagequi** [comment]
@tobiasKaminsky Ping ! :D

**Comment by Shagequi** [comment]
Hi, have some news about this one ? 
@AndyScherzinger @tobiasKaminsky ?

Thanks

**Comment by AndyScherzinger** [comment]
I am pretty swamped atm so I hope @tobiasKaminsky can take care of the review

File: src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 2]
```diff
@@ -2592,18 +2574,12 @@ public void onStart() {
                     startSyncFolderOperation(file, false);
                 }
             } else {
-                updateFragmentsVisibility(!file.isFolder());
-                updateActionBarTitleAndHomeButton(file.isFolder() ? null : file);
+                mLastDisplayedAccount = null;
             }
-        }
-        if (optionalUser.isPresent()) {
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java:2599]: Why is this removed?

**Comment by tobiasKaminsky** [comment]
FDA should not be changed at all for this fix, or?

**Comment by Shagequi** [comment]
No I don't think it need to be changed,
And I don't understand why  `if (optionalUser.isPresent()) { ` was removed, maybe a side-effect because the commits were done 4-5 months ago, and a new merge to remove conflict has been made to allow the merge of this pull request

About the indentation change, I'm sorry, that was before we discussed the importance of keeping the code in its original format ^^'

**Comment by tobiasKaminsky** [comment]
I removed FDA and will merge it after CI.

**Comment by Shagequi** [comment]
Perfect, tell me if you need more adjustements or precisions
